sqldelight-studio-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -52,6 +52,8 @@
   </module-components>
 
   <extensions defaultExtensionNs="com.intellij">
+    <compileServer.plugin classpath="sqldelight-studio-plugin-0.4.3-SNAPSHOT.jar"/>
```

Comment:
no. the recommended way to is to have a separate module called `sqldelight-jps-plugin` which has the server file / implementation and include it by doing 

`<compileServer.plugin classpath="sqldelight-jps-plugin.jar"/>`

So I could do that, but I wasn't sure how to have the jar packaged properly in with the studio plugin's jar, since we wouldn't want it to be version controlled.